lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1671,4 +1684,39 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
      )
    );
   }
+
+  void onCalendarEventReplyAction(EventAction eventAction) {
+    switch (eventAction.actionType) {
+      case EventActionType.yes:
+        _acceptCalendarEventAction();
+        break;
+      default:
+        break;
+    }
+  }
+  
+  void _acceptCalendarEventAction() {
+    if (_acceptCalendarEventInteractor == null) return;
+    if (_displayingEventBlobId == null) return;
+
+    consumeState(_acceptCalendarEventInteractor!.execute(
+      mailboxDashBoardController.accountId.value!,
```

Comment:
Please also make sure `accountId ` is not null

Reply:
Done